README.md

```diff
@@ -41,102 +41,76 @@ Python versions currently provided are:
 * [python-3.8](3.8)
 
 RHEL versions currently supported are:
-* RHEL7
+* RHEL 7 ([catalog.redhat.com](https://catalog.redhat.com/software/containers))
+* RHEL 8 ([catalog.redhat.com](https://catalog.redhat.com/software/containers))
 
 CentOS versions currently supported are:
-* CentOS7
+* CentOS 7 ([hub.docker.com](https://hub.docker.com/r/centos/))
 
+Fedora versions currently supported are:
+* Fedora 31 ([registry.fedoraproject.org](https://registry.fedoraproject.org/repo/f31/python3/tags/))
+* Fedora 32 ([registry.fedoraproject.org](https://registry.fedoraproject.org/repo/f32/python3/tags/))
 
-Installation
----------------
-To build a Python image, choose either the CentOS or RHEL based image:
-*  **RHEL based image**
-
-    These images are available in the [Red Hat Container Catalog](https://access.redhat.com/containers/#/registry.access.redhat.com/rhscl/python-36-rhel7).
-    To download it run:
-
-    ```
-    $ podman pull registry.access.redhat.com/rhscl/python-36-rhel7
-    ```
-
-    To build a RHEL based Python image, you need to run the build on a properly
-    subscribed RHEL machine.
-
-    ```
-    $ git clone https://github.com/sclorg/s2i-python-container.git
-    $ cd s2i-python-container
-    $ make build TARGET=rhel7 VERSIONS=3.6
-    ```
-
-*  **CentOS based image**
+Download
+--------
+To download one of the base Python images, follow the instructions you find in registries mentioned above.
 
-    This image is available on DockerHub. To download it run:
+For example, Centos image can be downloaded via:
 
-    ```
-    $ podman pull centos/python-36-centos7
-    ```
+```
+$ podman pull centos/python-36-centos7
+```
 
-    To build a Python image from scratch run:
+Build
+-----
+To build a Python image from scratch run:
 
-    ```
-    $ git clone https://github.com/sclorg/s2i-python-container.git
-    $ cd s2i-python-container
-    $ make build TARGET=centos7 VERSIONS=3.6
-    ```
+```
+$ git clone https://github.com/sclorg/s2i-python-container.git
+$ cd s2i-python-container
+$ make build TARGET=centos7 VERSIONS=3.6
+```
 
-Note: while the installation steps are calling `podman`, you can replace any such calls by `docker` with the same arguments.
```

Comment:
This is important NOTE for users who do not know the differences between podman / docker in detail. I would prefer to leave the Note here.

Reply:
The support for docker and podman is already mentioned in the very first paragraph but I have nothing against mentioning it here again.

Reply:
iirc the note was specifically asked for by @torsava in a previous PR

Reply:
I've added it to the top so it's one of the first things you see. And it's also the same for generated Readmes for Python versions.